web/chat/threads-tab.react.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+
+import * as React from 'react';
+
+type Props = {
+  title: string,
+  ...
+};
+export default function ThreadsTab(props: Props) {
+  return <div title={props.title} />;
```

Comment:
I don't think we render this

Reply:
yes, now with the approach you proposed, I've removed ThreadsTab since we don't need this